database/object_user_attributes/db.go

```diff
@@ -206,6 +210,46 @@ func (db *DB) GetObjectUserAttributesByObjectAttributeID(
 	return attributes, nil
 }
 
+func (db *DB) GetObjectUserAttributesByObjectIDsAttributeIDs(
+	ctx context.Context,
+	objectAttributeNames []string,
+	objectIDs []umid.UMID,
+	query string,
+	orderType universe.OrderType,
+	limit uint,
+	offset uint,
+) ([]*entry.ObjectUserAttribute, error) {
+	var searchString string
+	if query != "" {
+		searchString = fmt.Sprintf("%%%s%%", query)
+	}
+
+	sqlQuery := getObjectUserAttributesByObjectIDsAttributeIDsQuery
+	sqlParams := []interface{}{objectAttributeNames, objectIDs}
```

Comment:
In the model of this platform it's only 'unique' by plugin_id+attribute_name. So instead of objectAttributeNames array, a []entry.AttributeID (which encapsulates plugingID+attributeName) should be given as the input param of the function I think.